util/include/crypto_utils.hpp

```diff
@@ -15,68 +15,97 @@
 #include <utility>
 #include <string>
 #include <memory>
+
+#include <openssl/bio.h>
+#include <openssl/ec.h>
+#include <openssl/pem.h>
+#include <openssl/x509.h>
+#include <openssl/evp.h>
+
 namespace concord::util::crypto {
 enum class KeyFormat : std::uint16_t { HexaDecimalStrippedFormat, PemFormat };
 enum class CurveType : std::uint16_t { secp256k1, secp384r1 };
+
+class CertificateUtils {
```

Comment:
Do we need this class? Couldn't the functions simply sit in the `crypto` namespace? Or a new sub-namespace instead of the class - `concord::util::crypto::certs` or similar?